index.js

```diff
@@ -29,6 +29,62 @@ const redisStore = (...args) => {
         }
       })
     ),
+    mset: function(...args) {
+      const _this = this;
+
+      return new Promise((resolve, reject) => {
+        let cb;
+        let options = {};
+
+        if (typeof args[args.length - 1] === 'function') {
+          cb = args.pop();
+        }
+
+        if (args[args.length - 1] instanceof Object && args[args.length - 1].constructor === Object) {
+          options = args.pop();
+        }
+
+        if (!cb) {
+          cb = (err, result) => (err ? reject(err) : resolve(result));
+        }
+
+        const ttl = (options.ttl || options.ttl === 0) ? options.ttl : storeArgs.ttl;
+
+        let multi;
+        if (ttl) {
+          multi = redisCache.multi();
+        }
+
+        const length = args.length;
+        let key;
+        let value;
+        const parsed = [];
+        for (let i = 0; i < length; i += 2) {
+          key = args[i];
+          value = args[i + 1];
+
+          /**
+           * Make sure the value is cacheable
+           */
+          if (!_this.isCacheableValue(value)) {
+            return cb(new Error(`value cannot be ${value}`));
+          }
+
+          value = JSON.stringify(value) || '"undefined"';
+          parsed.push(...[key, value]);
+
+          if (ttl) {
+            multi.setex(key, ttl, value);
+          }
+        }
+
+        if (ttl) {
+          multi.exec(handleResponse(cb));
+        } else {
+          redisCache.mset.apply(redisCache, [...parsed, cb]);
```

Comment:
Shouldn't you use `handleResponse(cb)` here?

Reply:
Mmm yes you're right it should. Strange that the tests did not catch that. I'll make the change.

Reply:
It worked because the function signature of `handleResponse` is the same as [the one we set](https://github.com/dabroek/node-cache-manager-redis-store/pull/6/files#diff-168726dbe96b3ce427e7fedce31bb0bcR48) when a callback is not passed to `mset`.